### ISSUE TITLE
MongoDB support

### ISSUE THREAD

**Issue by hgeri** [keycloak/keycloak#26980]
### Description

Do you plan to get back Mongodb support? 

### Motivation

MongoDB supports multi-document transactions now.

**Comment by stianst** [comment]
No, we don't have capacity to maintain an integration with MongoDB
